downstream/modules/vault-aap/proc-vault-configuring-kv2-secret.adoc

```diff
@@ -0,0 +1,86 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="vault-configuring-kv2-secret"]
+
+= Configuring the `hashicorp.vault.kv2_secret` module
+
+[role="_abstract"]
+
+The `hashicorp.vault.kv2_secret` module performs Create, Update, and Delete operations on KV2 secrets through a unified interface.
+
+The corresponding `community.hashi_vault` modules are:
+
+* **`community.hashi_vault.vault_kv2_write`** - Write KV2 secrets.
+* **`community.hashi_vault.vault_kv2_delete`** - Delete KV2 secrets.
+
+.Prerequisites
+
+* Install the {PlatformNameShort} certified `hashicorp.vault` collection.
+
+.Procedure
+
+. Replicate your automation tasks from both of the `community.hashi_vault` modules to the following `hashicorp.vault.kv2_secret` parameters.  The `hashicorp.vault.kv2_secret` parameters are similar to `community.hashi_vault`. For examples, see _Migration examples for the `hashicorp.vault.kv2_secret` module_.
```

Comment:
Link missing here for  _Migration examples for the `hashicorp.vault.kv2_secret` module_ ?